pages/common/ifconfig.md

```diff
@@ -0,0 +1,23 @@
+# ifconfig
+
+> ifconfig - Interface Configurator, used to configure network interfaces.
+
+- View network settings of an ethernet adapter.
+
+`ifconfig eth0`
+
+- Display details of all interfaces, including disabled interfaces.
+
+`ifconfig -a`
+
+- Disable an interface.
+
+`ifconfig eth0 down`
+
+- Enable an interface.
```

Comment:
I'm sure most people will realize `eth0` is the interface, but it might be worth making this clearer, i.e, `Enable the eth0 interface`